### ISSUE TITLE
`yarn rw build web` requires `@redwoodjs/api-server`

### ISSUE THREAD

**Issue by jeliasson** [redwoodjs/redwood#2299]
**Issue**
In a serverful Docker deployment I have one `Dockerfile` for web and one `Dockerfile` for api. However, the web seems to require that the `api` directory is being copied in to the image in order to be able to run `yarn rw build web`.

**Example**
```
FROM node:14 as builder

COPY web web
COPY .nvmrc .
COPY babel.config.js .
COPY graphql.config.js .
COPY package.json .
COPY redwood.toml .
COPY yarn.lock .

# Install dependencies
RUN yarn install --frozen-lockfile

# Build
RUN yarn rw build web

# [...]
```
**Error**
![image](https://user-images.githubusercontent.com/865493/115012582-47be0400-9eb0-11eb-9f06-7c9506fe733c.png)

I think it's because the root package.jsons dependency to `@redwoodjs/cli` which has a dependency later to `@redwoodjs/api-server`. Not sure.

Anyone want to help me understand why this is, and how one can remove this dependency for web builds?

Thanks!

/cc @dac09 @peterp

**Comment by dac09** [comment]
Hey @jeliasson, v0.30 going out shortly that should address this. (fixed in #2251)

**Comment by dac09** [comment]
V0.30 Released! Please give it a go, and if it works for you, would appreciate if you close out the issue 

https://github.com/redwoodjs/redwood/releases/tag/v0.30.0

**Comment by jeliasson** [comment]
Unfortunately the same issue @dac09 

![image](https://user-images.githubusercontent.com/865493/115029002-ccffe380-9ec5-11eb-80cb-9954b113a2d7.png)

**Comment by dac09** [comment]
Was the lockfile updated after the upgrade? The may change was making sure the cli has api-server as a dependency.

**Comment by dac09** [comment]
Ah I just realised. No it doesn't.

Quick fix for now

```
yarn add -W --dev @redwoodjs/api-server
```
I'll fix this as part of the new serve in v0.31.

**Comment by jeliasson** [comment]
I have added `@redwoodjs/api-server` as dev depenency and it works fine. Let's keep this issue open and track fix in `0.30.0`. Sounds good? Thanks!
